src/chunk.c

```diff
@@ -1210,15 +1230,16 @@ append_chunk_oid(ChunkScanCtx *scanctx, Chunk *chunk)
 }
 
 static ChunkResult
-append_chunk(ChunkScanCtx *scanctx, Chunk *chunk)
+append_chunk(ChunkScanCtx *scanctx, ChunkStub *stub)
```

Comment:
```suggestion
append_chunk_stub(ChunkScanCtx *scanctx, ChunkStub *stub)
```

Reply:
No, this function appends the chunk, not the stub to the data field